aiven/client/cli.py

```diff
@@ -298,7 +298,13 @@ def get_project(self):
         """Return project given as cmdline argument or the default project from config file"""
         if getattr(self.args, "project", None) and self.args.project:
             return self.args.project
-        return self.config.get("default_project")
+        default_project = self.config.get("default_project")
```

Comment:
Are there any tests for this module? It would be useful to have this behavior tested

Reply:
Tests were added.